### ISSUE TITLE
Add a proper readme

### ISSUE THREAD

**Issue by OlivierJM** [geraldMaboshe/todoWithSub#2]
Just like the initial repo, the readme is better when it contains information that best reflect what the repo is all about, this makes it easy for other people to know what the repo is all about and how to use it.

This is not super important but when you have time, work on it.

**Comment by OlivierJM** [comment]
You can add the link from here https://github.com/geraldMaboshe/todoWithSub/issues/1#issuecomment-507360772 in the readme as well

**Comment by geraldMaboshe** [comment]
Noted.
On it.

**Comment by geraldMaboshe** [comment]
Done!
Check out the updated readMe. Waiting for your assessment feedback

**Comment by OlivierJM** [comment]
It looks great, here is a tip for more info you can add

- [ ]   How to run it on a local machine
- [ ]  Tools used (react, netlify, jsonplaceholder)
- [ ]  Inspiration 🤔🤷🏾‍♂️ if you will
- [ ]  More updates (consider this like a todo)

**Comment by geraldMaboshe** [comment]
Inspiration for coming up with the up?

What's there to talk about updates?

**Comment by OlivierJM** [comment]
Yes that would be the inspiration and on updates are like things you plan to work on in future like upgrades.
e.g: add a search filter, persist list items in a local storage, make it look more beautiful, etc ...

**Comment by geraldMaboshe** [comment]
Check out this one with how to run and tools used

**Comment by OlivierJM** [comment]
Nice, that is more content, it looks better
Here is another tip, did you know with Markdown you can represent code as code like this `this appears as code and it is highlighted`
You can use the above for things like `npm install ` or `npm install` to make them look more distinct from the rest of the text.

**Comment by OlivierJM** [comment]
And if you need more inspiration on how readmes look like try these 
- https://github.com/OlivierJM/react-github-api
- https://github.com/SparkEdUAB/sparked-server
- https://github.com/devconzm/devconzm

**Comment by OlivierJM** [comment]
Overall @geraldMaboshe I must say your willingness to learn and apply is unmatched 💪🏾.
Keep it up and am super glad to see that you made the upgrades as requested.

**Comment by geraldMaboshe** [comment]
Thank you so much @OlivierJM . Keep tagging me along.

**Comment by OlivierJM** [comment]
No worries bro, just let me know when you need help with what I can help with, I am more responsive on Github than elsewhere and If I happen to be busy then it is at the end of the day. Thanks 
I will be looking forward to seeing more updates

**Comment by geraldMaboshe** [comment]
Cool
